File: csharp/UrakawaToolkit/MediaLocation.cs
using System;

namespace urakawa.media
{
	/// <summary>
	/// MediaLocation is just a string which represents a file's path
	/// This simple idea could be extended in the (near)future.
	/// </summary>
	public class MediaLocation : IMediaLocation
	{
		public string Location;

		public MediaLocation()
		{
		}

		public MediaLocation(string loc)
		{
			Location = loc;
		}

		public IMediaLocation copy()
		{
			return new MediaLocation(Location);
		}
	}
}

File: trunk/java/UMLGraph/src/org/daisy/urakawa/media/MediaLocation.java
package org.daisy.urakawa.media;

import org.daisy.urakawa.xuk.XukAble;

/**
 * Abstract location. e.g.: URI
 */
public interface MediaLocation extends XukAble {
    /**
     * @return a distinct copy of the MediaLocation object.
     */
    MediaLocation copy();
}
